localstack/services/dynamodbstreams/provider.py

```diff
@@ -70,9 +70,19 @@ def describe_stream(
                 # Replace Kinesis ShardIDs with ones that mimic actual
                 # DynamoDBStream ShardIDs.
                 stream_shards = copy.deepcopy(stream_details["StreamDescription"]["Shards"])
-                for shard in stream_shards:
-                    shard["ShardId"] = shard_id(shard["ShardId"])
+                start_index = 0
+                for index, shard in enumerate(stream_shards):
+                    shard["ShardId"] = get_shard_id(stream_arn, shard["ShardId"])
                     shard.pop("HashKeyRange", None)
+                    # we want to ignore the shards before exclusive_start_shard_id parameters
+                    # we store the index where we encounter then slice the shards
+                    if exclusive_start_shard_id and exclusive_start_shard_id == shard["ShardId"]:
+                        start_index = index
+
+                if exclusive_start_shard_id:
+                    # slicing the resulting shards after the exclusive_start_shard_id parameters
+                    stream_shards = stream_shards[start_index + 1 :]
```

Comment:
I think this is Black formatting: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices
It's because there is the + operator on the left, it adds a space. It also weirded me out.

Reply:
My bad, you are right, totally discard this comment :D (although I really dislike Black's choice here xD)